### PR TITLE
Feature/finish off route nav

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/nih_wayfinding/.jshintrc [hunk 1]
```diff
@@ -19,6 +19,8 @@
   "smarttabs": true,
   "globals": {
     "angular": false,
+    "$": true,
+    "google": true,
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/.jshintrc:23]: Yay jshint fix

File: src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js [hunk 2]
```diff
@@ -193,17 +193,32 @@
...
+            Directions.get(origin, destination).then(function (geojson) {
+                ctl.map.geojson = angular.extend(ctl.map.geojson, {
+                    data: geojson
+                });
+                Navigation.setRoute(geojson);
+                Navigation.stepFirst();
+            }, function () {
+                NavbarConfig.set({
+                    title: 'You are off course and we were unable to reroute you.',
+                    subtitle: 'Tap "Reroute" to get new directions.',
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js]: Maybe confusing phrasing:  can user be re-routed, or not?

Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js]: Maybe subtitle should be 'Tap "Reroute" to manually choose your destination'?

**Comment by flibbertigibbet** [comment]
+1

File: src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js (hunk 2 above)
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js]: Sounds good.

**Pull request closed without merging**